shell/app/modules/cmp/pages/cluster-manage/add-cluster-forms/cluster-form.tsx

```diff
@@ -276,9 +275,13 @@ const ClusterAddForm = (props: any) => {
       />
       {clusterType === 'edas' ? null : (
         <div className="more">
-          <a className="more-btn" onClick={() => setShowMore(!showMore)}>
+          <a className="more-btn w-20 flex" onClick={() => setShowMore(!showMore)}>
```

Comment:
whether the width is large enough in en mode

Reply:
modified